README.md

```diff
@@ -30,7 +30,7 @@
 # 📖 [Awesome Boook] <a name="about-project"></a>
 
 
-**[Awesome Boook]** "Awesome books" is a simple website that displays a list of books and allows you to add and remove books from that list.
+**[Awesome Boook]** "Awesome books" is a simple website that displays a list of books and allows you to add and remove books from that list
 
```

Comment:
- [ ] As I requested in the previous review, could you please double-check if you've added the requested sections to your README file as per my previous review? It would be great if you could take a moment to ensure they're included.

Your table of content should contain:
```
# 📗 Table of Contents

- [📖 About the Project](#about-project)
  - [🛠 Built With](#built-with)
    - [Tech Stack](#tech-stack)
    - [Key Features](#key-features)
  - [🚀 Live Demo](#live-demo)
- [💻 Getting Started](#getting-started)
  - [Setup](#setup)
  - [Prerequisites](#prerequisites)
  - [Install](#install)
  - [Usage](#usage)
  - [Run tests](#run-tests)
  - [Deployment](#deployment)
- [👥 Authors](#authors)
- [🔭 Future Features](#future-features)
- [🤝 Contributing](#contributing)
- [⭐️ Show your support](#support)
- [🙏 Acknowledgements](#acknowledgements)
- [❓ FAQ (OPTIONAL)](#faq)
- [📝 License](#license)
```
   ### This is your actual README:
   ![image](https://github.com/jlberbesi/Awesome_books.github.io/assets/107952724/0afc519c-823b-4392-a5a4-35b94b1b1453)

   ### This is the [README template](https://github.com/microverseinc/readme-template), kindly check the content inside Getting started:
   ![image](https://github.com/jlberbesi/Awesome_books.github.io/assets/107952724/6d045f55-6a2c-426f-8b40-ffce1cb1bacb)